packages/webapp/pages/posts/[id]/index.tsx

```diff
@@ -124,7 +125,7 @@ const PostPage = ({ id, initialData }: Props): ReactElement => {
     options: { initialData, retry: false },
   });
   const containerClass = classNames(
-    'max-w-screen-laptop pb-20 laptop:min-h-page laptop:pb-6 laptopL:pb-0',
+    'mb-16 max-w-screen-laptop tablet:mb-8 laptop:mb-0 laptop:min-h-page laptop:pb-6 laptopL:pb-0',
```

Comment:
On mobile, the footer navbar is now much bigger because of the new comment button. Added bottom margin here so that the button does not cover the content.